pip/commands/install.py

```diff
@@ -127,6 +128,12 @@ def __init__(self, *args, **kw):
             help='Install using the user scheme.')
 
         cmd_opts.add_option(
+            '--global',
+            dest='use_user_site',
+            action='store_false',
+            help='Install into the global site packages.')
+
+        cmd_opts.add_option(
             '--egg',
```

Comment:
I don't know enough about how command options translate to environment variables and/or config file entries. Does this mean that ```PIP_GLOBAL``` or a ```global``` setting in the config file would make global the default (once the hard-coded default becomes "user")? If not, how *would* someone say they wanted to have global as the default?